config/logger.js

```diff
@@ -1,17 +1,29 @@
 import { serverEnv } from "@config/schemas/serverSchema";
 import * as PinoLogger from "pino";
 
-const config = {
+const development = {
   transport: {
     target: "pino-pretty",
     options: {
+      level: "debug",
+      colorize: true,
+      translateTime: "SYS:dd-mm-yyyy HH:MM:ss",
+      ignore: "pid,hostname", //Ignores process id and hostname
+    },
+  },
+};
+const production = {
+  transport: {
+    target: "pino-pretty",
+    options: {
+      level:'info',
```

Comment:
Not following coding style

```suggestion
      level: "info",
```